packages/core/src/features/table/table-feature.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getFeature } from "@k8slens/feature-core";
+import tableComponentInjectable from "./table-component.injectable";
+
+export const tableFeature = getFeature({
+  id: "core-table-feature",
+
+  register: (di) => {
+    di.register(tableComponentInjectable);
+  },
+});
```

Comment:
Can you remind me (mostly for learning purpose) why do we need to make it as a `feature`?
(instead of simple some injection token re-use from some package, e.g. `import {cystomTableToken} from "@k8slens/item-list")`?

Reply:
From my understanding, it seems to be an architectural decision. If we utilize an injection token in the core and there are two "users" of this token registered, an error will occur. Only one injection token consumer is allowed. However, by incorporating a feature within the open-lens package, we can easily separate features to determine their visibility in all lens variations. Additionally, the feature is beneficial when dealing with multiple injection tokens that need to be registered.

Reply:
@jansav @Iku-turso A good chance to clarify the question and correct me.

Reply:
> Additionally, the feature is beneficial when dealing with multiple injection tokens that need to be registered.

Can't get it.. why not `di.injectMany(customTableToken)`? And decide which one to use..  
This is simple & perfect for plain component injections/replacement).